Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/IncidentProducer/IncidentProducer.cs

```diff
@@ -0,0 +1,252 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using StackExchange.Redis;
+using System.Collections.Generic;
+using System.Collections;
+using System.IO;
+using System.Net.Http.Headers;
+using System.Net.Http;
+using System.Net;
+using System.Text.Json;
+using System.Text;
+using System.Threading.Tasks;
+using System;
+using System.Dynamic;
+using Newtonsoft.Json.Converters;
+using Microsoft.WindowsAzure.Storage;
+using Helios2Sentinel;
+
+namespace Helios2Sentinel
+{
+    public class IncidentProducer
+    {
+        private static readonly object queueLock = new object();
+        private static Lazy<ConnectionMultiplexer> lazyConnection = CreateConnection();
+        private static string containerName = Environment.GetEnvironmentVariable("containerName");
```

Comment:
There's no need to take it from the environment as it's created by deployment wizard and never changed. Set it equal to "cohesity-extra-parameters"

Reply:
so, you means this change?
-        private static string containerName = GetSecret("containerName");
+        private static string containerName = "cohesity-extra-parameters";

Reply:
Yes, but also can you move shared constants and functions to one place, so both producer and consumer can use them. For ex, you can create a shared library for it or reference from both projects.